packages/core/src/components/table/table.stories.js

```diff
@@ -351,9 +351,10 @@ export const VerticalHeaderAndLines = () => `
   </div>
 `;
 
-export const CustomHeaderBackgroundColorForDarkVariant = () => `
+export const CustomBackgroundColorsForDarkVariant = () => `
   <style type="text/css">
     .custom-background-color-1 {
+      --table-background-color: var(--color-black-5);
       --background-color: var(--color-tram);
```

Comment:
I guess it might be logical to rename this to header-background-color. But that would be a breaking change =/

Reply:
- [x] rename --background-color to --header-background-color